ibis/backends/duckdb/datatypes.py

```diff
@@ -139,3 +144,13 @@ def struct():
 )
 def parse_type(*args, **kwargs):
     return parse(*args, **kwargs)
+
+
+@to_sqla_type.register(DuckDBDialect, dt.UUID)
+def sa_duckdb_uuid(_, itype):
+    return postgresql.UUID
```

Comment:
How about just returning the toplevel `sa.UUID` instead?

```suggestion
    return sa.UUID()
```

Reply:
It is not supported in SQLAlchemy < 2.0.
https://github.com/sqlalchemy/sqlalchemy/blob/b3bb55aaea9ee94e50f8db594d65c2bfcdb314ef/lib/sqlalchemy/__init__.py#L89-L131